ports/mimxrt/boards/MIMXRT1020_EVK/mpconfigboard.mk

```diff
@@ -16,6 +13,19 @@ else
 JLINK_CONNECTION_SETTINGS =
 endif
 
+CFLAGS += \
+	-DUSDHC_AVAIL=1
+
+SRC_C += \
+	sdcard.c \
+	machine_sdcard.c \
+	hal/flexspi_nor_flash.c
+
+SRC_QSTR += \
+	machine_sdcard.c
```

Comment:
as above, this should not be added by a board but rather included in the Makefile unconditionally

Reply:
fixed.